packages/console-client/src/client/http/axios/middlewares/middlewareFactory.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable lines-around-comment */
+/* eslint-disable valid-jsdoc */
```

Comment:
are these rules needed?

Reply:
Yes these are needed as our eslint rules does not allow
- the first line of a code block to be blank
- some not blank lines above a comment block

thus are creating a loop on invalid lint


the second rule is because `@public` and `@static` are not recognized as a valid JSDoc for some reason